tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cc

```diff
@@ -1066,7 +1064,7 @@ TEST (nnstreamer_nnfw_mlapi, multimodal_01_p)
       ML_PIPELINE_BUF_POLICY_DO_NOT_FREE);
   EXPECT_EQ (status, ML_ERROR_NONE);
 
-  g_usleep (5000000);
+  g_usleep (500000); /* wait for sink callback */
```

Comment:
5000ms to 500ms?

Reply:
Since other cases wait for 500ms, IMO, 5secs are too long for this case. Is it too short?

Reply:
This depends on what is it waiting for. We'd better add some status check loops instead of excessive sleep.

Reply:
Actually current nnfw unit-test has no case to check the data received in sink node.
It just checks data type only in the sink callback.